corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -191,7 +194,14 @@ def expand_group(group):
     def expand_location_ids(domain, location_ids):
         user_ids = set()
         for location_id in location_ids:
-            for user in get_all_users_by_location(domain, location_id):
+            if toggles.INCLUDE_ALL_LOCATIONS.enabled(domain):
+                user_ids_at_this_location = user_ids_at_locations([location_id])
+                users = [CouchUser.wrap_correctly(u)
+                         for u in iter_docs(CouchUser.get_db(), user_ids_at_this_location)]
```

Comment:
Nit, looks like the rest of the adjacent code uses lazy evaluation, so you might as well do so here too.  Could save some memory if it's handled properly by the callers.
```suggestion
                users = (CouchUser.wrap_correctly(u)
                         for u in iter_docs(CouchUser.get_db(), user_ids_at_this_location))
```
or 
```suggestion
                users = map(CouchUser.wrap_correctly,
                            iter_docs(CouchUser.get_db(), user_ids_at_this_location))
```